UnitTest/UnitTestTests/StringExtensionTests.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using UnitTest;
+
+namespace UnitTestTests
+{
+    [TestClass]
+    public class StringExtensionTests
+    {
+        [TestMethod]
+        public void Handle_When_ColorIsBlack_Then_MustBeTrue()
```

Comment:
Why Handle, if your action is IsBaseColor?

Reply:
Better to name it as `IsBaseColor_When_ColorIsBlack_Then_ShouldReturnTrue`

Reply:
Done in 3b0d4f72e4804b22ea2b3b5437831f9e099c7dd2